e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -72,6 +73,21 @@ describe('I18456 Built-in slash commands: common', () => {
             cy.get(`#postMessageText_${postId}`).should('have.text', 'test ¯\\_(ツ)_/¯');
         });
     });
+
+    it('MM-T2345 /me on RHS', () => {
+        loginAndVisitDefaultChannel(user1, testChannelUrl);
+        cy.postMessage(MESSAGES.MEDIUM);
+
+        // # Open RHS (reply thread)
+        cy.clickPostCommentIcon();
+
+        // # type /me test
+        cy.get('#reply_textbox').type('/me test');
+        cy.get('#addCommentButton').click();
+
+        // * Properly formatted with lower opacity
+        cy.get('div[id="postListContent"]').contains('p', 'test').should('be.visible');
```

Comment:
It should also verify that the last post is from the current user.

Reply:
I've added `cy.apiGetPost()` and use it in the test to verify the author of the post.

Reply:
Okay, I think it's fine though I'd prefer verifying it via UI.